build.gradle

```diff
@@ -128,6 +128,10 @@ java {
     withJavadocJar()
 }
 
+jar {
```

Comment:
I don't think we want these example classes in the jar file. These should be treated in a similar way as tests. Also, given their names they might be confusing. e.g. `TaskRunner`

![image](https://user-images.githubusercontent.com/4755315/200423796-6c66598c-0e0c-4055-8507-774eaff0c917.png)

Reply:
Also, why is the package `io.orkes.conductor.sdk.examples` when the main package of this project is `io.orkes.conductor.client`?

Reply:
I think this is prob copy/paste?  We should make it all consistent in naming.